### DISCUSSION TITLE
Using the full screen in a quarto document

### DISCUSSION THREAD

**Discussion by PhilSmith26** [quarto-dev/quarto-cli#3885] [Q&A] [answered]
Is there a simple way to use the full screen width in a quarto document? My documents use only the middle one third. For example see https://rpubs.com/PhilSmith26/941490

**Comment by mcanouil** [comment, marked as answer]
See <https://quarto.org/docs/authoring/article-layout.html> and <https://quarto.org/docs/prerelease/1.3.html>.

**Reply by PhilSmith26** [reply]
Thanks mcanouil.

It worked after I updated to the quarto pre-release 1.3 and added
format:
  html: 
    grid:
      sidebar-width: 0px
      body-width: 2000px
      margin-width: 0px
      gutter-width: 1.5rem
to the YAML
